.github/workflows/static.yaml

```diff
@@ -43,4 +43,4 @@ jobs:
 
     - name: Run docs and check extensions
       run: |
-        tox -e docs
+        tox -e testdocs
```

Comment:
I've seen differences between systemwide dictionaries between distros. So a spell check failure would not be reproducible. That would not be a major issue, since it's easy to fix by adding words to the `spelling_wordlist.txt`, but it would be an annoyance.

If you think it's too likely that broken spelling gets integrated otherwise, we could include it in CI though.

Reply:
well... if `testdocs` is not integrated into CI there is the risk of PR author and PR reviewer to forget to manually it.

What we can do, is have `tox -e testdocs` on the release checklist and execute it as part of the release job. 

This means, that spelling errors will be deferred and fixed when a release is done.

I prefer to run it on each PR...at least to give it a try and see if it's an annoyance.

I guess that you will see the errors in the GitHub Job logs and fix them right way, without needing to run it a again on your local system.